webrtc-stats.html

```diff
@@ -472,15 +473,27 @@ <h3>
             </p>
           </dd>
           <dt>
-            <dfn><code>track</code></dfn>
+            <dfn><code>sender</code></dfn>
           </dt>
           <dd>
             <p>
-              Contains statistics related to a specific MediaStreamTrack and the corresponding
+              Contains statistics related to a specific sender and the corresponding
```

Comment:
Here and elsewhere: prefer RTCRtpSender/RTCRtpReceiver. I realize this is a mix of moved text and new text though so we can do it separately if need be.